meson_options.txt

```diff
@@ -3,5 +3,6 @@ option('disable_drm', type : 'boolean', value : false)
 option('with_x11', type : 'combo', choices : ['yes', 'no', 'auto'], value : 'auto')
 option('with_glx', type : 'combo', choices : ['yes', 'no', 'auto'], value : 'auto')
 option('with_wayland', type : 'combo', choices : ['yes', 'no', 'auto'], value : 'auto')
+option('with_win32', type : 'combo', choices : ['yes', 'no', 'auto'], value : 'auto')
 option('enable_docs', type : 'boolean', value : false)
-option('enable_va_messaging', type : 'boolean', value : true)
+option('enable_va_messaging', type : 'boolean', value : true)
```

Comment:
Please tweak your editor settings and/or git config so these "No newline at end of file" are resolved.